framework/src/auxkernels/MaterialStdVectorAux.C

```diff
@@ -19,11 +19,14 @@ InputParameters validParams<MaterialStdVectorAux>()
 {
   InputParameters params = validParams<MaterialStdVectorAuxBase<> >();
   params.addClassDescription("Extracts a component of a material type std::vector<Real> to an aux variable.  If the std::vector is not of sufficient size then zero is returned");
+  params.addParam<unsigned int>("special_qp", "Evaluate the std::vector<Real> at this quadpoint.  This only needs to be used if you are interested in a particular quadpoint in each element: otherwise do not include this parameter in your input file");
```

Comment:
OK, that sounds fine - i don't know what an "Advanced" input parameter is though!